services/app-api/src/postgres/contractAndRates/submitContract.ts

```diff
@@ -159,10 +173,7 @@ async function submitContract(
             return await findContractWithHistory(tx, contractID)
         })
     } catch (err) {
-        console.error('Prisma error submitting contract', err)
```

Comment:
did you get a type error here or something? Technically err is unknown so your change is more correct but I've never seen something complain